src/main/java/fr/uge/chargepointconfiguration/chargepoint/ocpp/OcppVersion.java

```diff
@@ -18,6 +19,7 @@ public enum OcppVersion {
   public static Optional<OcppVersion> parse(String version) {
     return switch (version) {
       case "ocpp1.6" -> Optional.of(V1_6);
+      case "ocpp2.0.1" -> Optional.of(V2);
       default -> Optional.empty();
```

Comment:
I would raise an exception here, not Optional.empty(), not sure, we sould ask @Idhao

Reply:
Let us continue coding, this is not really important for now...